timesketch/frontend-ng/src/components/LeftPanel/Search.vue

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <template>
-  <div>
+  <div v-if="iconOnly" class="pa-4" style="cursor: pointer" @click="$emit('toggleDrawer')">
```

Comment:
When on another canvas (e.g. Analyzer or Graph) and clicking the search icon in the minimized version it just toggles the left panel but does not trigger the canvas to change.

Reply:
Good catch. Fixed now.